aat/main_test.go

```diff
@@ -51,12 +46,41 @@ func TestMain(m *testing.M) {
 	//router.DebugEnabled = true
 	router.SetLogger(rtrLogger)
 
+	crAuth, err := auth.NewCRAuthenticator(&testCRAuthenticator{})
+	if err != nil {
+		panic(err)
+	}
+
 	// Create router instance.
-	nxr = router.NewRouter(autoRealm, strictURI)
-	nxr.AddRealm(wamp.URI(testRealm), anonAuth, allowDisclose)
+	routerConfig := &router.RouterConfig{
+		RealmConfigs: []*router.RealmConfig{
+			&router.RealmConfig{
+				URI:           wamp.URI(testRealm),
+				StrictURI:     false,
+				AnonymousAuth: true,
```

Comment:
I have not used it, but that's not to say that my use case is the usual case. I didn't make a change not, but we can if we decide that that's the way to proceed.